README.md

```diff
@@ -359,12 +359,21 @@ A few things to check before you go live:
 - Make sure you have set up webhooks to receive live events
 - Make sure you have entered correct billing details inside your [Onfido Dashboard](https://onfido.com/dashboard/)
 
-## Browser support
+## More information
+
+### Browser compatibility
 
 ![Chrome](https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png) | ![Firefox](https://raw.githubusercontent.com/alrra/browser-logos/master/src/firefox/firefox_48x48.png) | ![IE](https://raw.githubusercontent.com/alrra/browser-logos/master/src/edge/edge_48x48.png) | ![Safari](https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari/safari_48x48.png)
 --- | --- | --- | --- |
 Latest ✔ | Latest ✔ | 11+ ✔ | Latest ✔ |
 
+### Support
+
+Please open an issue through [GitHub](https://github.com/onfido/onfido-sdk-ui/issues). Please be as detailed as you can. Remember **not** to submit your token in the issue. Also check the closed issues to check whether it has been previously raised and answered.
```

Comment:
` Also check the closed issues to check` sounds a bit redundant. Would one of these two options read better? `Also check the closed issues to find out [...]` or `Also check the closed issues to make sure that the issue hasn't been previously raised and answered`

Reply:
thanks. fair comment. I copied this from iOS. However, to be honest, at this stage, i think I will keep it as it is, as the other two PRs have been approved.